static/js/settings_notifications.js

```diff
@@ -46,6 +52,34 @@ function rerender_ui() {
     } else {
         unmatched_streams_table.css("display", "table-row-group");
     }
+
+    // Has to be in rerender_ui because relevant rows in table are added there
+    // therefore event handler has to be set up after every call of rerender_ui
+    unmatched_streams_table.find(".unmute_stream").on("click", (e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        const row = $(e.currentTarget).closest(".stream-row");
+        const stream_id = Number.parseInt(row.attr("data-stream-id"), 10);
+        const sub = sub_store.get(stream_id);
+
+        stream_settings_ui.set_muted(
+            sub,
+            !sub.is_muted,
+            row.closest(".subsection-parent").find(".alert-notification"),
+        );
+
+        $(row).toggleClass("control-label-disabled", false);
+        $(row).find(".unmute_stream").hide();
+        $(row).find("input[disabled]").prop("disabled", false);
+
+        const mobile_notifications_disabled =
+            settings_config.all_notifications(user_settings).show_push_notifications_tooltip
+                .push_notifications;
+
+        if (mobile_notifications_disabled) {
+            $(row).find('[name="push_notifications"]').prop("disabled", true);
```

Comment:
Would something like `$(row).find('[name="push_notifications"]').prop("disabled", !page_params.realm_push_notifications_enabled);` be more readable here?